apps/balancer-tools/src/app/apr/round/(components)/PoolsCards.tsx

```diff
@@ -10,52 +21,125 @@ export interface PoolData {
   apr: number;
 }
 
-function PoolCard({
-  data: { symbol, pct_votes: pctVotes, votes, apr },
+export function PoolCard({
+  data: { symbol },
+  roundId,
 }: {
-  data: PoolData;
+  data: DuneGaugeData;
+  roundId?: string;
 }) {
+  // TODO: Decision. Some pools are not going to have gauges associated to them.
+  // Or the gauges are not whitelisted yet.
+  // We should not display them. Ideally not even fetch them.
+  let gauge;
+  try {
+    gauge = new Gauge(symbol);
```

Comment:
Hadn't looked into those classes, great stuff.

If "symbol" is the response from the API, it won't have the network attribute.